src/appengine/handlers/cron/cleanup.py

```diff
@@ -703,16 +731,35 @@ def _send_email_to_uploader(testcase_id, to_email, content):
   mail.send(to_email, subject, html_content)
 
 
-def _update_issue_severity_labels(testcase, issue):
+def _get_severity_from_labels(security_severity_label, labels_lowercase):
+  """Get the severity from the label list."""
+  pattern = issue_filer.get_label_pattern(security_severity_label)
+  for label in labels_lowercase:
```

Comment:
why can't we do the .lower() here instead of caller ?

Reply:
No reason. This was copied from label_utils. Changed.